.github/workflows/build.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
           sudo du -hs / 2> /dev/null || true
 
       - name: tarball build
-        run: tar czvf riscv.tar.gz -C /opt/ riscv/
+        run: XZ_OPT="--threads=0 -6e" tar cJf riscv.tar.xz -C /opt/ riscv/
```

Comment:
Just curious - how were the `XZ_OPT` options chosen? 
Why `--threads=0`? Does "parallel" processing cause problems with the resulting output?
Isn't `-6` compression level the default anyway so why specify it explicitly?

- https://linux.die.net/man/1/xz#:~:text=%2D6%20...%20%2D9,and%20memory%20usage.

Could the `XZ_OPT` settings be specified once and reused elsewhere so that if they ever need to be tweaked it is just a single line change?

Reply:
When T=0 xz uses  all available cores/hardware threads, -e means extreme (so it's not 6 which is the default but 6e).